src/pages/About.tsx

```diff
@@ -9,28 +9,47 @@ export default function About() {
+                                For the future of Khoj, we imagine building a rich ecosystem of integrations to support use-cases we haven't imagined yet.
+                                And having your personal assistant always available.
                             </p>
                             <p className='content'>
-                                <b>Khoj is 100% open source</b>, and we welcome contributions from the community. Being open source is by design. We believe that this ecosystem will grow best with active involve from the community. We also believe that the best way to build trust is to be transparent.
+                                <b>Khoj is open source by design</b>.
+                                Only with transparency and trust can we build a thriving community.
+                                Only with a thriving community can we build an ecosystem that serves the diverse, unique needs of everyone.
                             </p>
                         </div>
                         <h2 className='article-title'>Why we're making it</h2>
                         <div className='content'>
                             <p className='content'>
-                                Over the last 3-4 decades, zetabytes of data have moved online into the internet. While people have quickly adopted use of the internet, the tools to access and use this data have not kept up. The next wave of innovation will be in tools that help people access and use this data in a way that's useful to them, while improving their productivity by 10x.
+                                We've seen zetabytes of data moving online over the last 3-4 decades.
+                                But the tools to access and use this data have not kept up, especially for personal data.
+                                The current wave of AI innovations are creating tools that will vastly improve the ability of people to use this data.
                             </p>
                             <p className='content'>
-                                With the recent breakthroughs in Artificial Intelligence, there's no reason why we should settle for the current state of tools. As this technology matures, it's critical that everyone have access to these tools for productivity and creativity, without giving up their privacy.
+                                We have the opportunity to guide this wave of innovation.
+                                To let everyone benefit from the productivity and creativity gains from these tools, without giving up their privacy.
+                            </p>
+                            <p className='content'>
+                                <b>We're trying to democratize access to these tools. </b>
+                                So that the productivity gains from these advances are available to the most diverse range of people possible.
```

Comment:
Make sure to avoid fragments (aka incomplete sentences).

```suggestion
                                <b>We're trying to democratize access to these tools. </b>, so that the productivity gains from these advances are available to the most diverse range of people possible.
```